eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Branching"
+        },
+        "definition": {
+          "en": "Branching scenarios involve the learner making decisions and following different paths based on those decisions."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/feedback",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Feedback"
+        },
+        "definition": {
+          "en": "Feedback activities allow the learner to submit feedback to the course instructor."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/imageMap",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Image map"
+        },
+        "definition": {
+          "en": "Image map defines an interface wherein users must correctly identify portions of an image by tagging them via image overlay."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/matching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Matching"
+        },
+        "definition": {
+          "en": "Learners must match labeled items to the correct bucket."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/multipleChoice",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Multiple choice"
+        },
+        "definition": {
+          "en": "A single question with multiple answers for the learner to choose from."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/sortable",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Sortable"
+        },
+        "definition": {
+          "en": "A single question with multiple answers that must be sorted in the correct order."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/videoBranching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Video branching"
+        },
+        "definition": {
+          "en": "The learner watches a video and is then asked a question. Depending on how they answer, they are taken to another video or another question."
+        }
+      }
+    ],
+    "templates": [
```

Comment:
Since you're using Statement Templates, you'll need to use Patterns, too. If you don't want to express the rules for ordering Statements in this version of the profile (though I notice a lot of implicit ones in the naming of the statement templates, so it might be worthwhile), an easy way to do that is a single pattern with a `oneOrMore` that points to another pattern with an `alternates` attribute that has every statement template. That combination just says "a session is one or more statements that are any of these statements".

Reply:
I included a patterns array with the two patterns specified.